scripts/scrape-neo-images.ts

```diff
@@ -0,0 +1,120 @@
+import fetch from "node-fetch";
+import * as cheerio from "cheerio";
+import * as fs from "fs";
+import * as path from "path";
+import { kNeoDatasetConfigs } from "../src/models/neo-datasets";
+
+interface ImageInfo {
+  date: string;
+  id: string;
+}
+
+interface DatasetInfo {
+  images: ImageInfo[];
+  maxResolution: {
+    width: number;
+    height: number;
+  };
+}
```

Comment:
These should be imported from `neo-datasets.ts`.

Reply:
I'll address this in the next PR.